### DISCUSSION TITLE
Amplify hosting breaks when using node-linker=hoisted in .npmrc

### DISCUSSION THREAD

**Discussion by jeroenroumen-acc** [vercel/turborepo#5233] [Help]
So trying to deploy a Next 13 application to AWS Amplify with SSR. They officially have monorepo support but so far it's been proven quite difficult. Been going through their guide https://github.com/aws-amplify/amplify-hosting/blob/main/FAQ.md#support-for-monorepo-framework which mentions the need for adding the `node-linkder=hoisted` to the `.npmrc` when using `pnpm`.

When dependencies are installed I get various errors like below in the `_app.tsx` and `_document.tsx` files. So adding the option for the node_modules hosting creates some TypeScript issues.
`'Component' cannot be used as a JSX component `

Does anyone else encounter something like this?

**Comment by anthonyshew** [comment]
This doesn't sound like it's in the Turborepo domain so I'm going to close this one up. If anyone out there has the answer, though, please chime in!
